File: semestr_2/lista3_lab/zad2/Szesciokat.cpp
#include "Szesciokat.hpp"

double Szesciokat::obliczPole()
{
    return 3 * bok * bok * std::sqrt(3) / 2;
}
double Szesciokat::obliczObwod()
{
    return 6 * bok;
}

File: semestr_2/lista3_lab/zad1/Szesciokat.java
import java.lang.Math;  

public class Szesciokat implements Figura {
    public double obliczObwod()
    {
        return 6 * bok;
    }
    public double obliczPole()
    {
        return 3 * bok * bok * Math.sqrt(3) / 2;
    }

    public Szesciokat(double _bok)
    {
        bok = _bok;
    }

    private double bok;
}
